pkg/eks/api.go

```diff
@@ -263,10 +244,6 @@ func New(spec *api.ProviderConfig, clusterSpec *api.ClusterConfig) (*ClusterProv
 		provider.elbv2 = elbv2.New(s, s.Config.Copy().WithEndpoint(endpoint))
 
 	}
-	if endpoint, ok := os.LookupEnv("AWS_STS_ENDPOINT"); ok {
```

Comment:
This should probably stay though? And should be added to STSV2.... Right?

Reply:
Yep, this should use

```go
	// An endpoint resolver that can be used to provide or override an endpoint
	// for the given service and region.
	//
	// When EndpointResolverWithOptions is specified, it will be used by a
	// service client rather than using EndpointResolver if also specified.
	//
	// See the `aws.EndpointResolverWithOptions` documentation for additional
	// usage information.
	EndpointResolverWithOptions EndpointResolverWithOptions
```

Reply:
Oh dang, this isn't nice. :/ https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/endpoints/#overriding-endpoint-with-fallback

Reply:
Turned out to be fine.